microsoft/PowerToys
# Mouse Utils - Mouse Highlighter

## Summary of the Pull Request

**What is this about:**
Adds a new PowerToy called "Mouse Highlighter", under "Mouse Utilities".
This PowerToys can be used to highlight left and right mouse clicks while presenting/screen sharing.
Default activation key is Windows+Shift+H.

![image](https://user-images.githubusercontent.com/26118718/142655209-65ea177b-e85c-4f40-ba69-0b969eab08b5.png)


**What is include in the PR:** 
- The code for the PowerToys.
- Settings page and Oobe description line.
- Logging and telemetry.

![image](https://user-images.githubusercontent.com/26118718/142655316-b91f22f5-b654-43a0-badf-b649f07e2f32.png)

**How does someone test / validate:** 
Start PowerToys with the module enabled and press "Windows"+"Shift"+"H". Then left and right click with the mouse to check the effect. "Windows"+"Shift"+"H" to toggle it off again.

## Quality Checklist

- [x] **Linked issue:** Mostly discussed in our meetings, but can also be an initial implementation for #13808
